### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by Memorablea** [Memorablea/COVID-19-#45] [Q&A]
@FedeGueli @ryhisner  Sorry to disturb you, I am a beginner of COVID-19 mutation. I can't find your contact information and want to ask some questions, so I put forward this here.

- S:T385I(C22716T)

**Comment by FedeGueli** [comment]
Hi! no worries. Which is the problem with that? i saw a S:T385I from NZl last week and something yesterday mentioned by someone somewhere but i didnt pay so much attention. Is it a growing mutation??

**Reply by Memorablea** [reply]
What about S: T346I?

**Comment by FedeGueli** [comment]
![Schermata 2023-07-10 alle 16 11 50](https://github.com/Memorablea/COVID-19-/assets/87669813/dfb8f863-5d02-4610-879a-e3844f521965)
![Schermata 2023-07-10 alle 16 12 17](https://github.com/Memorablea/COVID-19-/assets/87669813/3d66c6c1-aa50-41a8-98f9-7a2bb7c6c041)

**Reply by Memorablea** [reply]
It's nothing, just discovered in some suspected persistent infections and transition evolution.
